### PR TITLE
#160528936 ft-fetch-single-order 

### PR DESCRIPTION
**Pull request by Johnpaul-Borree**
#### What does this PR do?
Create Api endpoint to allow users fetch single order
#### Description of Task to be completed?
- Create a controller action for this feature
- Validate for invalid Id
- Create a route to controller
#### How should this be manually tested?
Test endpoint by starting server and navigating to GET /api/v1/orders/ordersId
#### What are the relevant pivotal tracker stories?
#160528936
#### Screenshots (if appropriate)
screenshot 17

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ]
+    }
+}
```
Comment (KaiserPhemi) [inline .eslintrc.json]: #### Nitpick
New line

File: test/ordersTests.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+				.end((err, res) => {
+					res.should.have.status(200);
+					res.body.should.be.a('object');
+					res.body.should.have.property('Orders').be.a('array');
+					res.body.should.have.property('status').eql('Success');
+					done();
+				});
+		});
+	});
+});
```
Comment (KaiserPhemi) [inline test/ordersTests.js]: #### Nitpick
New line

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
Implement feedback

File: .eslintrc.json (hunk 1 above)
Comment (Johnpaul-Borree) [reply .eslintrc.json]: finished implementing this feedback just now, thanks

File: test/ordersTests.js (hunk 2 above)
Comment (Johnpaul-Borree) [reply test/ordersTests.js]: implemented, thanks

**Pull request merged**